pkg/cmd/prebuild/add.go

```diff
@@ -134,8 +152,11 @@ var prebuildAddCmd = &cobra.Command{
 	},
 }
 
-var runOnAddFlag bool
-
 func init() {
 	prebuildAddCmd.Flags().BoolVar(&runOnAddFlag, "run", false, "Run the prebuild once after adding it")
+	prebuildAddCmd.Flags().StringVar(&branchFlag, "branch", "", "Git branch for the prebuild")
+	prebuildAddCmd.Flags().IntVar(&retentionFlag, "retention", 0, "Retention period for the prebuild (in days)")
+	prebuildAddCmd.Flags().IntVar(&commitIntervalFlag, "commit-interval", 0, "Commit interval for the prebuild (in seconds)")
+	prebuildAddCmd.Flags().StringSliceVar(&triggerFilesFlag, "trigger-files", nil, "Files that trigger the prebuild")
+	prebuildAddCmd.Flags().StringVar(&projectConfigFlag, "project-config", "", "Project configuration name")
```

Comment:
`project-config` shouldn't be a flag but an argument of the command. `add [PROJECT_CONFIG]` and then extract that from `args`

Reply:
modified the project-config as the args